CMakeLists.txt

```diff
@@ -419,6 +419,7 @@ include_directories(depends/tthread)
 include_directories(${ZLIB_INCLUDE_DIRS})
 include_directories(depends/clsocket/src)
 include_directories(depends/xlsxio/include)
+include_directories(depends/imgui)
```

Comment:
will plugin code be using this API directly, or will it just be through the DFHack module? If the latter, we should probably restrict the include to target that directly depend on it. Something like:
```
target_include_directories(imgui INTERFACE depends/imgui)
```
(this would have to appear below the `add_subdirectory(imgui)` line below so the target exists

Reply:
I spent a while fixing up the C++ support, so the current design should directly allow for plugins to use ImGui directly without many restrictions. I do need to add some testing, but yes ImGui itself would be directly available to any plugins as long as they also decorate their render/feed code correctly from the module itself